This patch is supposed to be used in conjonction with
document.execCommand("enableObjectResizing", false, false);
However it is either fixed, or is only needed when dragging two consecutive images,
see https://github.com/ProseMirror/prosemirror/issues/692 for link to upstream chrome issue.
--- node_modules/prosemirror-view/dist/index.js	2017-12-15 22:02:14.981764871 +0100
+++ node_modules.new/prosemirror-view/dist/index.js	2017-12-18 00:38:17.760292981 +0100
@@ -997,7 +997,7 @@
       var assign;
       ((assign = prosemirrorModel.DOMSerializer.renderSpec(document, node.type.spec.toDOM(node)), dom = assign.dom, contentDOM = assign.contentDOM));
     }
-    if (!contentDOM && !node.isText) {
+    if (!contentDOM && !node.isText && dom.nodeName != "IMG") {
       dom.contentEditable = false;
       if (node.type.spec.draggable) { dom.draggable = true; }
     }
